doc/projectBranding.md

```diff
@@ -102,3 +107,137 @@ Finally, if you'd rather not display any "About" pop-up and remove the associate
 // import { AboutContent } from '../../components/AboutContent';
 export const About = null;
 ```
+
+# Translations customization
+
+In order to edit translation labels, you can directly edit translation files :
+[public/locales/en/translation.json](../public/locales/en/translation.json) or/and
+[public/locales/fr/translation.json](../public/locales/fr/translation.json).
+
+More convenient, you can set translation custom files who override translation.json :
+[public/locales/en/custom.json](../public/locales/en/custom.json) or/and
+[public/locales/fr/custom.json](../public/locales/fr/custom.json).
+
+By default, these json files are filled with empty object
```

Comment:
I'm not sure we need to explicit it, `{}` is the default for an empty json, isn't it ?

Reply:
A big refacto has been done by @csm-thu